5.3.1-SNAPSHOT build failing validation locally

## Required Information

* [X] AEM Version, including Service Packs, Cumulative Fix Packs, etc: N/A
* [X] ACS AEM Commons Version: 5.3.1-SNAPSHOT
* [X] Reproducible on Latest? yes



@kwin last night i tried doing a build from `master` (5.3.1-SNAPSHOT) and my build failed w/ validation errors (See below). Im a little confused since this certainly worked when cutting the 5.3.0 release.

IIRC you changed some of the filters, etc. - anything here jump out at you as to why its failing? Could some external validation deps have changed? 

Doubly odd the dame code seems to build in Github actions...

When doing a `mvn clean package` i am getting validation errors:


```
[INFO] --- filevault-package-maven-plugin:1.3.0:validate-files (default-validate-files) @ acs-aem-commons-ui.content ---
[INFO] Using META-INF/vault from /Users/davidg/Code/acs-aem-commons/ui.content/src/main/content/META-INF/vault
[INFO] Validate files in generatedMetaInfRootDirectory 'target/vault-work/META-INF' and metaInfRootDir 'target/vault-work/META-INF'
[INFO] Using 12 validators for package of type CONTENT: jackrabbit-accesscontrol (org.apache.jackrabbit.vault.validation.spi.impl.AccessControlValidator), jackrabbit-filter (org.apache.jackrabbit.vault.validation.spi.impl.AdvancedFilterValidator), jackrabbit-properties (org.apache.jackrabbit.vault.validation.spi.impl.AdvancedPropertiesValidator), jackrabbit-docviewparser (org.apache.jackrabbit.vault.validation.spi.impl.DocumentViewParserValidator), netcentric-aem-classification (biz.netcentric.filevault.validator.aem.classification.AemClassificationValidator), netcentric-aem-cloud (biz.netcentric.filevault.validator.aem.cloud.AemCloudValidator), jackrabbit-dependencies (org.apache.jackrabbit.vault.validation.spi.impl.DependencyValidator), jackrabbit-duplicateuuid (org.apache.jackrabbit.vault.validation.spi.impl.DuplicateUuidValidator), jackrabbit-emptyelements (org.apache.jackrabbit.vault.validation.spi.impl.EmptyElementsValidator), jackrabbit-mergelimitations (org.apache.jackrabbit.vault.validation.spi.impl.MergeLimitationsValidator), jackrabbit-packagetype (org.apache.jackrabbit.vault.validation.spi.impl.PackageTypeValidator), jackrabbit-nodetypes (org.apache.jackrabbit.vault.validation.spi.impl.nodetype.NodeTypeValidator)
[INFO] Scanning baseDir 'src/main/content/META-INF'...
[INFO] Scanning path 'src/main/content/META-INF'...
[INFO] Scanning path 'src/main/content/META-INF/vault'...
[INFO] Scanning path 'src/main/content/META-INF/vault/filter.xml'...
[INFO] Scanning baseDir 'target/vault-work/META-INF'...
[INFO] Scanning path 'target/vault-work/META-INF'...
[INFO] Scanning path 'target/vault-work/META-INF/MANIFEST.MF'...
[INFO] Scanning path 'target/vault-work/META-INF/vault'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/config.xml'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/definition'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/filter.xml'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/nodetypes.cnd'...
[INFO] ValidationViolation: "jackrabbit-nodetypes: CDN file vault/nodetypes.cnd registered for node type validation", filePath=target/vault-work/META-INF/vault/nodetypes.cnd
[INFO] Scanning path 'target/vault-work/META-INF/vault/properties.xml'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/settings.xml'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/definition/.content.xml'...
[INFO] Scanning path 'target/vault-work/META-INF/vault/definition/thumbnail.png'...
[INFO] Scanning baseDir 'src/main/content/jcr_root'...
[INFO] Scanning path 'src/main/content/jcr_root'...
[INFO] Scanning path 'src/main/content/jcr_root/conf'...
[INFO] Scanning path 'src/main/content/jcr_root/content'...
[INFO] Scanning path 'src/main/content/jcr_root/etc'...
[INFO] Scanning path 'src/main/content/jcr_root/home'...
[INFO] Scanning path 'src/main/content/jcr_root/var'...
[INFO] Scanning path 'src/main/content/jcr_root/conf/global'...
[INFO] Scanning path 'src/main/content/jcr_root/content/_rep_policy.xml'...
[ERROR] ValidationViolation: "jackrabbit-filter: Node '/content/rep:policy/allow' is not contained in any of the filter rules", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow, line=7, column=48
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:privileges' [Name] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow, line=7, column=48
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:principalName' [String] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow, line=7, column=48
[ERROR] ValidationViolation: "jackrabbit-filter: Node '/content/rep:policy/allow0' is not contained in any of the filter rules", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow0, line=11, column=48
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:privileges' [Name] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow0, line=11, column=48
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:principalName' [String] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow0, line=11, column=48
[ERROR] ValidationViolation: "jackrabbit-filter: Node '/content/rep:policy/allow1' is not contained in any of the filter rules", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow1, line=15, column=83
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:privileges' [Name] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow1, line=15, column=83
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:principalName' [String] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow1, line=15, column=83
[ERROR] ValidationViolation: "jackrabbit-filter: Node '/content/rep:policy/allow2' is not contained in any of the filter rules", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow2, line=19, column=113
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:privileges' [Name] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow2, line=19, column=113
[WARNING] ValidationViolation: "jackrabbit-nodetypes: Property 'rep:principalName' [String] is not allowed in node with potential default types [nt:folder]: No applicable property definition found for name and type!", filePath=src/main/content/jcr_root/content/_rep_policy.xml, nodePath=/content/rep:policy/allow2, line=19, column=113
[ERROR] ValidationViolation: "jackrabbit-filter: Node '/content/rep:policy/allow1' is not contained in any of the filter rules", filePath=src/main/content/jcr_root/content/_rep_policy.xml
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 <!-- Keep this up to date! After a release, change the tag name to the latest release -->
 [unreleased changes details]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-5.0.14...HEAD
 
+### Fixed
+
+- #2826 - 5.3.1-SNAPSHOT build failing validation locally
+
 ## 5.3.0 - 2022-04-15
 
 ### Fixed

--- a/pom.xml
+++ b/pom.xml
@@ -233,6 +233,7 @@ Bundle-DocURL: https://adobe-consulting-services.github.io/acs-aem-commons/
                         <group>adobe/consulting</group>
                         <filterSource>src/main/content/META-INF/vault/filter.xml</filterSource>
                         <thumbnailImage>src/main/resources/thumbnail.png</thumbnailImage>
+                        <failOnUncoveredSourceFiles>true</failOnUncoveredSourceFiles>
                         <validatorsSettings>
                             <jackrabbit-filter>
                                 <options>

--- a/ui.content/src/main/content/META-INF/vault/filter.xml
+++ b/ui.content/src/main/content/META-INF/vault/filter.xml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
+    <filter root="/content/rep:policy"/>
     <filter root="/content/dam/rep:policy"/>
     <filter root="/conf/global/settings">
         <include pattern="/conf/global/settings/redirects"/>

